examples/dune

```diff
@@ -0,0 +1 @@
+(dirs :standard \ aides_logement)
```

Comment:
Why?

Reply:
Ah this is temporary and will be removed from the finalised PR, see  0a98f6331aaf283838f78fcc8536ae29f2ba5c55